umm-spec-lib/src/cmr/umm_spec/migration/service_service_options.clj

```diff
@@ -2,7 +2,8 @@
   "Contains functions for migrating between versions of the UMM Service schema
    for service options."
   (:require
-   [clojure.set :as set]))
+   [clojure.set :as set]
+   [cmr.common.util :refer [remove-nil-keys remove-empty-maps]]))
```

Comment:
Nitpick: is there a reason you're using `refer` here?
```suggestion
   [cmr.common.util :as util]))
```

Reply:
Since these are the only 2 functions that are being used, I didn't think that I needed to load or get the entire library. But I can just use as.